src/components/Badge/Badge.tsx

```diff
@@ -13,8 +13,13 @@ export interface BadgeProps extends TPAComponentProps {
   priority?: BADGE_PRIORITY;
 }
 
+export interface DefaultProps{
```

Comment:
I don't think we need this to be exported.
And It causes the build to fail.
So you can just remove the `export`.